buzzmobile/process/gps_mapper/gps_mapper.py

```diff
@@ -16,16 +16,25 @@
 # 'bearing' is counter-clockwise angle from north in radians.
 # 'points' will contain the polyline list of lat-lon points
 # (normalized to some size based on pixels_per_m).
-# 'location' is our lat-lon position (unmodified) from our GPS.
+# 'fixes' is a list of our last 'median_filter_size' number of lat-lon
+# coordinates which are passed through a median filter
+# to reduce the effect of noise in our location data.
 g['bearing'] = g['points'] = g['location'] = None
+g['fixes'] = []
 # These ranges are km dimensions of the path. Initialize them to 0.
 # Also initialize the scaled width and height of the points to 0.
 g['y_range'] = g['x_range'] = g['height'] = g['width'] = 0
+# These are the lat-lon dimensions that are used
+# to normalize the lat-lon points (initalized to 0).
+g['ll_height'] = g['ll_width'] = 0
+# Initialize the top left and bottom right image coordinates to (0, 0).
+g['top_left'] = g['bottom_right'] = (0, 0)
```

Comment:
The others are all good, but this one is treading dangerous waters. If this were a list and not a tuple, this would be a bug waiting to happen, I'd break this into 2 statements.